src/lib/util/util.ts

```diff
@@ -575,7 +590,7 @@ export function ratelimit(bucket: number, cooldown: number, auth = false) {
 
 			try {
 				bucket.drip();
-			} catch {}
+			} catch { }
```

Comment:
Why? Linter error or?

Reply:
Might've been VSCode's autoformatter as well. I didn't actively change it.